server/src/main/java/org/apache/druid/server/coordinator/loading/HttpLoadQueuePeon.java

```diff
@@ -192,26 +168,23 @@ private HistoricalLoadingCapabilities fetchSegmentLoadingCapabilities()
       ).get();
 
       if (HttpServletResponse.SC_NOT_FOUND == responseHandler.getStatus()) {
-        log.info(
+        log.warn(
             "Historical capabilities endpoint not found at server[%s]. Using default values.",
-            new URL(serverId)
+            segmentLoadingCapabilitiesURL
         );
-        return new HistoricalLoadingCapabilities(1, 1);
-      }
-
-      if (HttpServletResponse.SC_OK != responseHandler.getStatus()) {
+        return new SegmentLoadingCapabilities(1, 1);
```

Comment:
I wonder what we should do in a case where a node is in turbo-mode but the API call to fetch capabilities has failed.

In that case, I feel that we should use the `batchSize` (for both normal and turbo) if it has been configured and is higher than 1.